src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ContentPageViewModel.cs

```diff
@@ -79,6 +79,9 @@ private void FetchContent()
             throw;
         }
 
+        var oneContent = newContent.Count == 1;
+        newContent.ForEach(c => c.OnlyControlOnPage = oneContent);
```

Comment:
_clever_

Reply:
though why is this better than `if (newContent.Count == 1) { newContent[0].OnlyControlOnPage = true; }` ???

Reply:
![speechless-finger](https://github.com/user-attachments/assets/01fee645-759e-485f-85b9-898a1820e8bc)